src/gui/src/gui.cpp

```diff
@@ -37,6 +37,7 @@
 #include <boost/algorithm/string/predicate.hpp>
 #include <stdexcept>
 #include <string>
+#include <tcl.h>
```

Comment:
do you need this or just a forward decl of 
extern "C" {
struct Tcl_Interp;
}

Reply:
I can check on that, but you are probably right that it's not needed since nothing else is using Tcl in gui.cpp.